package.json

```diff
@@ -1,7 +1,11 @@
 {
   "license": "Apache-2.0",
   "scripts": {
-    "prepare": "husky"
+    "prepare": "husky",
+    "start": "./support-frontend/devrun.sh", 
+    "setup":  "./support-frontend/setup.sh",
```

Comment:
I'm not sure this adds a lot of value, it will only be used once per laptop install so just having the shell script seems fine then we can keep this file minimal